ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
Why remove these optional traces? They don't print unless you define the variable? Also this is not the only place they are used.